packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -669,9 +669,11 @@ class ScrollbarPainter extends ChangeNotifier implements CustomPainter {
   /// The thumb's corresponding scroll offset in the track.
   double getThumbScrollOffset() {
     final double scrollableExtent = _lastMetrics!.maxScrollExtent - _lastMetrics!.minScrollExtent;
+    final double maxFraction = _lastMetrics!.maxScrollExtent / scrollableExtent;
+    final double minFraction = _lastMetrics!.minScrollExtent / scrollableExtent;
```

Comment:
is is possible for both min/max scroll extent to be negative/positive infinite as well as scrollable Extent?

Reply:
At least for one case, when the scroll extent is infinite the scrollbar is disabled.

Reply:
gotcha, in that case, I think we should either handle it in code or assert if we are confident that is this method won't be called when it is infinite